### PR TITLE
Feature#4/creat views

### PR DESCRIPTION
**Pull request by claunicole**
# Description
What did you implemented/Why did you implemented this:
 -Created view of the employees
 -Created view of the company branch
 -Created the check application view in Home
 -Added bootstrap gem 
 -Styled the home view 

 # Testing
 Help me how can I test or look at the changes
-Create a new company branch in rails console, now run rails server and enter the /companies route in your browser, check if you can see the name of the branch.

 ## Screenshots
Home_view

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -0,0 +1,124 @@
+body {
```
Comment (dannharo) [inline app/assets/stylesheets/home.scss]: please sort all properties on your `scss` files

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  color: #ffffff;
+}
+.button_login {
+  position: absolute;
+  width: 178px;
+  height: 60px;
+  left: 950px;
+  top: 65px;
+
+  background: #4895ef;
```
Comment (dannharo) [inline app/assets/stylesheets/home.scss]: uses colors as [variables](https://sass-lang.com/documentation/variables) to make changes easier.

Comment (dannharo) [reply app/assets/stylesheets/home.scss]: I recommend you read about [The 7-1 Pattern](https://sass-guidelin.es/#the-7-1-pattern)
for a better management of your site styles.

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<div>
```
Comment (dannharo) [inline app/views/home/index.html.erb:1]: check indentation here

File: app/views/employees/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<h1>Employees#index</h1>
+<p>Find me in app/views/employees/index.html.erb</p>
```
Comment (dannharo) [inline app/views/employees/index.html.erb]: you could avoid this also [here](https://github.com/BC-MAY-21-ROR/kata-06-el-reloj-checador-k6-ca-reilero-01/pull/10/files#diff-4ddbd659dee033dbee675502845ec3de63686d5bf5156f752b5dc39ad5767a06R2)

File: app/views/companies/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<h1>Companies#index</h1>
+<p>Find me in app/views/companies/index.html.erb</p>
+<% @companys.each do |company| %>
+
+<p><%= company.name %></p>
```
Comment (dannharo) [inline app/views/companies/index.html.erb]: indentation into the `do` block

File: app/assets/stylesheets/sass/utils/_colors.scss [hunk 6]
```diff
@@ -0,0 +1,8 @@
+:root{
+--black: black;
+--blue: #4895ef;
+--dark-grey: rgba(0, 18, 25, 0.96);
+--light-grey: #44565d;
+--white: #faf9f9;
+--yellow:  #ffc300;
+}
```
Comment (dannharo) [inline app/assets/stylesheets/sass/utils/_colors.scss]: variables ca be defined like just like this:
```
$dark-grey: rgba(0, 18, 25, 0.96);
```

and use it:
```
color: $dark-grey;
```
instead:
```
color: var(--dark-grey);
```

as they mention [here](https://sass-lang.com/documentation/variables)

File: app/views/home/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
```
Comment (dannharo) [inline app/views/home/index.html.erb:1]: indentation here, in the other files you're using two spaces.

File: app/assets/stylesheets/application.css [hunk 8]
```diff
@@ -12,4 +12,7 @@
  *
  *= require_tree .
  *= require_self
+ *= require app
+ 
+
```
Comment (dannharo) [inline app/assets/stylesheets/application.css]: you could delete these empty lines

**Review by dannharo — COMMENTED** [review]
looks good, just some minor comments

**Review by YilKanda — CHANGES_REQUESTED** [review]
Solve the reviews of @dannharo

File: app/assets/stylesheets/home.scss (hunk 1 above)
Comment (claunicole) [reply app/assets/stylesheets/home.scss]: Done

File: app/assets/stylesheets/home.scss (hunk 2 above)
Comment (claunicole) [reply app/assets/stylesheets/home.scss]: Done

File: app/views/home/index.html.erb (hunk 3 above)
Comment (claunicole) [reply app/views/home/index.html.erb:1]: Done

File: app/views/employees/index.html.erb (hunk 4 above)
Comment (claunicole) [reply app/views/employees/index.html.erb]: Done

File: app/assets/stylesheets/sass/utils/_colors.scss (hunk 6 above)
Comment (claunicole) [reply app/assets/stylesheets/sass/utils/_colors.scss]: Done

**Review by dannharo — COMMENTED** [review]
check [this file](https://github.com/BC-MAY-21-ROR/kata-06-el-reloj-checador-k6-ca-reilero-01/pull/10/files#diff-70acf32a949fd60a5a3d1ae8bbca9090058a7f011f201c41e6a8b1a51ea12c02R1) indentation, only that, after that ping me to approve it

File: app/views/home/index.html.erb [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+		<div class="form">
+			<%= form_for :post do |f| %>
+				<%= f.text_field :title, placeholder:'Número privado', class:'text-field'%><br>
+				<div class="button-container">
+					<%= f.submit 'Check', class:'button'%>
+				</div>
+			<%end%>
+		</div>
+	</div>
+</div>
```
Comment (dannharo) [inline app/views/home/index.html.erb:14]:
```suggestion
<div>
  <div class= "watch_container">
    <h4 class="h4"> <%= Time.now.strftime("%F")%> </h4>
    <h5 class="h5">Hora <%= Time.now.strftime("%H:%M:%S")%> </h5>
    <div class="form">
      <%= form_for :post do |f| %>
        <%= f.text_field :title, placeholder:'Número privado', class:'text-field'%><br>
        <div class="button-container">
          <%= f.submit 'Check', class:'button'%>
        </div>
      <%end%>
    </div>
  </div>
</div>
```

**Pull request merged**